gc/gc-tool/src/main/java/org/projectnessie/gc/tool/cli/options/MarkOptions.java

```diff
@@ -36,6 +36,7 @@
 import picocli.CommandLine.ParameterException;
 
 public class MarkOptions {
+  @SuppressWarnings("JavaTimeDefaultTimeZone")
```

Comment:
nit: maybe add a comment to explain that we are intentionally using the CLI users timezone here (and it will not get mixed with server timezone later on)